samcli/commands/list/exceptions.py

```diff
@@ -0,0 +1,16 @@
+"""
+Exceptions for SAM list
+"""
+
+
+from samcli.commands.exceptions import UserException
+
+
+class NoRegionError(UserException):
```

Comment:
In `samcli/commands/exceptions.py` we have a `RegionError` that we can reuse.

Reply:
Replaced it with RegionError